.github/workflows/continuous-build.yml

```diff
@@ -1,19 +1,33 @@
-name: Continuous Build Server
+name: Continuous Build
+
+# Trigger on pushes or pull requests to the master branch, but not for documentation-only updates.
 on:
   push:
     branches:
       - master
+    paths-ignore:
+      - '**/*.md'
   pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**/*.md'
+
+# DEFAULT_OS is ubuntu-20.04 (focal) as that's the same as we use in Travis. This is find/replaced
```

Comment:
Not sure, but are you able to use a single-element item in the matrix as a variable?

```
matrix:
  ubuntu-os:
  - ubuntu-20.04

...

if: matrix.os == ${{ matrix.ubuntu-os }}
```

Reply:
will try it! right now GH aren't charging us per run so why not give it a shot :D

Reply:
I think the last commit works. but GH actions is being annoying. will try to kick it with close/reopen